examples/secure-tracing/otel-collector-config.yaml

```diff
@@ -0,0 +1,26 @@
+extensions:
+  health_check:
+
```

Comment:
We can also remove the configuration definition:

```suggestion
```

Reply:
My bad, missed this. Removed.